tests/unit_system_test.py

```diff
@@ -111,6 +111,13 @@ def test_expression_after_magnitude_conversion_in_compound_assignment(self):
         printed_rhs_expression = print_rhs_of_first_assignment_in_update_block(model)
         self.assertEqual(printed_rhs_expression, '0.001 * (1200*mV)')
 
+    def test_expression_after_magnitude_conversion_in_compound_mult_div(self):
+        model = ModelParser.parse_model(
+            os.path.join(os.path.realpath(os.path.join(os.path.dirname(__file__), 'resources')),
+                         'CompoundMultDivWithDifferentButCompatibleUnits.nestml'))
+        printed_rhs_expression = print_rhs_of_first_assignment_in_update_block(model)
+        self.assertEqual(printed_rhs_expression, '1200*mV')
```

Comment:
Isn't it better to add a coco test that errors out with such compound expressions?

Reply:
This is already covered by the type checking coco fixed by this PR. This tests checks that the coco picks up on it. I added a negative in addition to the positive test case to improve test coverage.